site/docs/guide/introduction.md

```diff
@@ -129,9 +130,48 @@ Please choose what you think is best for you.
 
 ### Prerequisites for Deno
 
-[Install Deno](https://deno.land/#installation) if you have not done that already.
-When you have created your bot, for example in a file called `bot.ts`, you can run it via `deno run --allow-net bot.ts`.
-You can stop it again with `Ctrl+C`.
+Before you can start creating your bot, let's first spend a few minutes on a proper setup for software development.
+This means installing a few tools.
+
+#### Preparing Your Machine for Development
+
+[Install Deno](https://deno.land/manual/getting_started/installation#download-and-install) if you have not done it already.
+
+You also need a text editor that fits well with coding.
+The best one for Deno is Visual Studio Code.
+[Install it.](https://code.visualstudio.com/)
+
+Next, you need to connect VSCode and Deno.
+That's very simple: There is an extension for VSCode that does everything automatically.
+You can install it [as described here](https://marketplace.visualstudio.com/items?itemName=denoland.vscode-deno).
+
+Your system is now ready for bot development! :tada:
+
+#### Developing a Bot
+
+Create a new directory somewhere.
+It will contain your bot project.
+Open this new directory in VSCode.
+
+```sh
+mkdir ./my-bot
+cd ./my-bot
+code .
+```
+
+> If you’re on macOS and the `code` command is not available, just open VSCode, hit `Ctrl+Shift+P`, type "shell command", and hit enter.
```

Comment:
Do you have any idea how this works on Windows?

Reply:
No. I wouldn’t bother tho. There’s something called PoawaerShall if I’m not mistaken. It should probably be there. Otherwise let them use WSL, or grab a VMWare torrent if they’re on Windows 7 or older.

Reply:
Alright, then let's not think about it

Reply:
Happy :)